examples/conway.rs

```diff
@@ -218,6 +228,28 @@ impl Config for Game {
                             client.send_message("I said left click, not right click!".italic());
                         }
                     }
+                    ClientEvent::InteractWithEntity { kind, .. } => {
+                        if kind == InteractWithEntityKind::Attack {
+                            if client.state.resource_pack_active {
+                                client.send_message("You already have the resource pack!".italic());
+                                continue;
+                            }
+
+                            client.set_resource_pack(
+                                "https://download942.mediafire.com/j0ag4kfe1ftg/0ds8poaabjc8snw/pack.zip"
```

Comment:
Let's avoid pinging mediafire all the time. If you upload the pack as an attachment here, I think you'll get a usable URL.

Reply:
Should the resource pack go in `assets/`?

Reply:
Yeah, that would probably be better. We could just have the resource pack in the repo. For this PR, that will need to land first so we can have the correct URL from github.

Reply:
I'll just leave the URL as localhost in the new example, and open another PR to change it when the pack.zip has been merged into main.